0a.md

```diff
@@ -0,0 +1,80 @@
+NIP-0a
+======
+
+Audio Events
+---------------
+
+`draft` `optional`
+
+This NIP defines several new event kinds representing different types of audio track. These events
+are _parameterized replaceable_ as defined in [NIP-01](./01.md) and deletable per [NIP-09](09.md).
```

Comment:
Being replaceable events, I assume there is no way to change the pubkey (author) of future events, since replaceable events are dependent on the original event's pubkey. 

I'm thinking through the problematic scenario of publishing audio tracks under a general Music Feed pubkey, and later supporting an Artist's ability to self-publish their audio tracks using their own pubkey, if/when they are onboarded to nostr and take custody of their own private key.

I suppose the older audio track event can be deleted when this happens, leaving the new Artist's self-published event? In this case, should the `d` tag be re-used or randomly re-generated?

Reply:
Correct, in that scenario you'd probably want to just delete the events and publish new ones. D tag persistence shouldn't matter because the pubkey would be changing anyhow.